.github/workflows/playwright.yml

```diff
@@ -0,0 +1,31 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Download freighter wallet extension
+        run: curl -L https://github.com/stellar/freighter/releases/download/5.27.1/build-5.27.1.zip -o dapp/e2e/freighter.zip
```

Comment:
One of the drawbacks is that dependabot is not going to catch that. But I guess it's fine for that purpose if we cannot use a normal deps.